mscr-ui/src/common/components/schema-info/index.tsx

```diff
@@ -0,0 +1,320 @@
+import * as React from 'react';
+import { useEffect } from 'react';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { SearchInput } from 'suomifi-ui-components';
+import IconButton from '@mui/material/IconButton';
+import ExpandLessIcon from '@mui/icons-material/ExpandLess';
+import Box from '@mui/material/Box';
+import SchemaTree from '@app/common/components/schema-info/schema-tree';
+import NodeInfo from '@app/common/components/schema-info/schema-tree/node-info';
+import { RenderTree } from '@app/common/interfaces/crosswalk-connection.interface';
+import { cloneDeep } from 'lodash';
+import { generateTreeFromJson } from '@app/common/components/schema-info/schema-tree/schema-tree-renderer';
+import { useGetFrontendSchemaQuery } from '@app/common/components/schema/schema.slice';
+
+const inputData: RenderTree[] = [];
```

Comment:
What's the purpose of setting this, instead of just setting the intial state as [] in the useState hooks?

Reply:
That variable is redundant. Possibly leftovers from earlier refactoring.